pysgrid/tests/test_sgrid.py

```diff
@@ -398,22 +393,14 @@ def test_node_dimesnions(sgrid_wrf_sgrid_2d):
     assert node_dims == expected
 
 
-# TestSGridSaveNodeCoordinates
-"""
-Test that SGrid.save_as_netcdf is saving
-content correctly.
+# Test that SGrid.save_as_netcdf is saving
+# content correctly.
 
-There maybe a better way to do this using
-mocks, but this will do for now.
-
-"""
-
-
-def round_trip_deltares(deltares_sgrid):
+def test_round_trip_deltares(deltares_sgrid):
```

Comment:
I would remove this from your PR to avoid mixing context. I am still working on another PR that re-factors the tests and improves the round-trip. However, it is fine by me if you want to leave this as is. I just need to rebase it later.